frontend/src/fetch-tva.js

```diff
@@ -33,12 +33,7 @@ import { extractSirenSlugFromUrl, formatIntFr } from './utils';
         // We dont log errors, as they are already logged in the backend
         tvaContainer.setError();
         if (e instanceof TypeError) {
-          throw new new FetchRessourceException({
-            ressource: 'VerifyTVA',
-            cause: e,
-            context: { siren },
-            administration: EAdministration.VIES,
-          })();
+          throw e;
```

Comment:
I would rather instantiate a new error to prevent the multiple duplicates in sentry due to browser specificity.

```suggestion
          throw new Error("Client error while fetching TVA", {cause : e});
```

Reply:
Seems good to me